libs/react-components/community/src/components/form/pickers/datepicker/datepicker.tsx

```diff
@@ -135,6 +136,10 @@ export const DatePicker = (props: DatePickerProps): JSX.Element => {
     onClose?.();
   };
 
+  // Mui doesn't pick correctly which component to render, thus it was unusable when zooming in on bigger screens
+  const isMobileLayout = useLayout(['mobile']);
```

Comment:
I get the need behind this change, but I would implement it through `desktopModeMediaQuery` prop https://mui.com/x/api/date-pickers/date-picker/ and possibly add a prop on our end so we could override it on the application level if needed.

Also wondering why MUI team decided this to be based on a touch device and not screen size...

Reply:
Missed that prop. Changed to use that